profiles/logical/holo/hydra/master/default.nix

```
@@ -109,6 +115,17 @@ in
     };
   };
 
+  services.restic.backups = {
+    remotebackup = {
+      user = "root";
+      paths = [ "/var/backup/postgresql/" ];
+      s3CredentialsFile = "/var/lib/hydra/pgbackup/keys/hydra-master-pgbackup/secret";
```

Let's place this in `/etc/nixos`. Hate that it's all scattered in random places across FS.

```suggestion
      s3CredentialsFile = "/etc/nixos/hydra-master-restic-credentials";
```